### ISSUE TITLE
Create Pools with team names only when tournament starts

### ISSUE THREAD

**Issue by Joe2k** [india-ultimate/hub#180]
Currently in Tournament Manager, Creating pools is tied to seedings. Thus when pools are created we cant edit the standings later. So we need to decouple these

**Comment by SibiAkkash** [comment]
The matches are given team names only when starting the tournament right (`start_tournament` in server/api) ?

If the seeding is updated after pools are created, the seed values are still the same, so the match name is still the same. Only the team names will change, which are only assigned while starting the tournament.

So they are already decoupled ? correct me i'm wrong lol

**Comment by Joe2k** [comment]
Hehe nope.. So when creating a Pool we are creating initial_seeding as {seed : team_id}

This is the prob in refering to.

So ideally we can just keep it like {seed : 0} And then populate team_id when we start the tournament based on the tournament initial seeding.

**Comment by SibiAkkash** [comment]
ah yes makes sense. 

In the pool creation UI, i'm using `tournamentSeeding` and `teamsMap` to show the team names. So when we update the seeding, the team names in the pools will also be updated. 

But, they will not get updated in the pool object's `initial_seeding` field in the DB. 

So while starting tournament, we can just update all pools to have the latest tournament seeding right ?

**Comment by SibiAkkash** [comment]
So probably we should store team IDs in the pool then ? Till we start tournament, we don't need the team names right ?

**Comment by Joe2k** [comment]
> So probably we should store team IDs in the pool then ? Till we start tournament, we don't need the team names right ?

This I didnt get you.. So ideally till we start tournament we dont need team IDs we just need the seed. And with seed we can get the team ID from tournament standings for the UI. only when we create the Pool we need to add the team_ids to the respective seeds

**Comment by SibiAkkash** [comment]
Lol I meant to write " we shouldn't store team IDs in the pool". I forgot that we need the team IDs to get the pool outcome

> ah yes makes sense.
> 
> In the pool creation UI, i'm using `tournamentSeeding` and `teamsMap` to show the team names. So when we update the seeding, the team names in the pools will also be updated.
> 
> But, they will not get updated in the pool object's `initial_seeding` field in the DB.
> 
> So while starting tournament, we can just update all pools to have the latest tournament seeding right ?

So this can be a fix ?

**Comment by Joe2k** [comment]
Yep thats what I was saying too.. We store just 0 as team_id when we create pool. But store the actual team_id only when starting the tournament
